modules/builtin/brightness.go

```diff
@@ -81,6 +81,14 @@ func (b *brightness) StatusAreaWidget() fyne.CanvasObject {
 	return fyne.NewContainerWithLayout(layout.NewBorderLayout(nil, nil, brightnessIcon, nil), brightnessIcon, bright)
 }
 
+func UpdateBrightness(value int) {
```

Comment:
I'm not really a fan of a module looking up itself to access a private method.
It feels like maybe such a lookup helper could be in tthe window manager code somewhere so this module doesnt need to know about desktop instances...